elements/pl-pivot-table/pl-pivot-table-double.mustache

```diff
@@ -0,0 +1,106 @@
+{{#question}}
+        <script>
```

Comment:
javascript should not be inline.  any separate .js files need to be in the element's directory and should be loaded via `script src=`.

Reply:
Is inline-js not allowed at all? I have seen built-in pl-element, pl-order-block is using inline-js. I just wonder whether there is any potential issue from inline-js.

https://github.com/PrairieLearn/PrairieLearn/blob/1e3deb438b2c2b95c6291811afe918f7144a3935/apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache#L16

Reply:
This is resolved